test_utilities/src/utilities.rs

```diff
@@ -26,6 +26,20 @@ use serde::{Deserialize, Serialize};
 extern crate time;
 use time::Instant;
 
+pub fn context() -> Context {
+    let execution_id = execution_id();
+    let home_dir = std::env::var("WORKSPACE_ROOT_DIR").unwrap_or(home_dir().unwrap().to_str().unwrap().to_string());
+    let lib_root_dir = std::env::var("LIB_ROOT_DIR").expect("LIB_ROOT_DIR is mandatory");
+
+    let metadata = Metadata {
+        dry_run_deploy: Option::from(false),
+        resource_expiration_in_seconds: Some(3600),
+        docker_build_options: Some("--network 'host'".to_string()),
```

Comment:
Maybe it can be good to add as a default settings even in prod ?
We are doing also dind so it can help 
What do you think ?

Reply:
At the moment there is no need in prod for this, only for tests purpose. May be later when the need will be there